Sources/ComposableArchitecture/Reducer/Reducers/CombineReducers.swift

```diff
@@ -26,7 +26,12 @@ public struct CombineReducers<Reducers: ReducerProtocol>: ReducerProtocol {
   public init(
     @ReducerBuilderOf<Reducers> _ build: () -> Reducers
   ) {
-    self.reducers = build()
+    self.init(internal: build())
+  }
+
+  @usableFromInline
+  init(internal reducers: Reducers) {
+    self.reducers = reducers
   }
```

Comment:
same situation here, but since the `init` takes an argument I can disambiguate by giving it a different external argument name.